docs/releasing.md

```diff
@@ -25,35 +43,108 @@ Release manager follows the below _Release Process_.
 
 ### Versioning
 
-We follow [semver](https://semver.org/) versioning semantics (`vMajor.Minor.Patch`), to align with the greater Node community. Generally, this means our bi-weekly releases will bump a minor. Though we will release a new patch version if high-priority fixes are required before the next schedule release. Additionally, if a schedule release contains no new features, then we'll only bump the patch version.
+We follow [semver](https://semver.org/) versioning semantics (`vMajor.Minor.Patch`). Breaking changes will bump the major version. New features or bug fixes will bump the minor version. If a release contains no new features, then we'll only bump the patch version.
 
 
 ## Release Process
 
+Note: You'll wanna be on a Linux machine, since the Lightrider step will require that.
+
+### On the scheduled release date
+
+Before starting, you should announce to the LH eng channel that you are releasing,
+and that no new PRs should be merged until you are done. If you have to go through all these steps
+again to assert that release-blocking fixes work, other PRs may be merged too, but that should
+be coordinate with you.
+
 ```sh
-# Run the tests
+# Run the tests.
 bash ./lighthouse-core/scripts/release/test.sh
+# Change into the pristine folder.
+cd ../lighthouse-pristine
+```
+
+Confirm DevTools integration will work:
+```sh
+# You should have Chromium already checked out at ~/chromium/src
+# See: https://www.chromium.org/developers/how-tos/get-the-code
+
+# Roll to Chromium folder.
+yarn devtools
+
+# Checkout latest Chromium code.
+cd ~/chromium/src
+git pull
+git new-branch lh-roll-x.x.x
+gclient sync
+autoninja -C out/Release chrome blink_tests
+
+# Run tests and rebase.
+yarn --cwd ~/chromium/src/third_party/blink/renderer/devtools test 'http/tests/devtools/audits/*.js' --reset-results
+# Verify the changes are expected.
+git diff
+
+# Verify that the Audits panel still works. Consider the new features that have been added.
+# If anything is wrong, stop releasing, investigate, and prioritize landing the PR.
+
+# For bonus points, add some tests covering new features. Either a new test, or an extra
```

Comment:
if this is a priority we should figure out a way to have this done when new features land. Putting it here means either we'll never release because no one will want to take on the massive work or this will never be done :)

Reply:
New features usually come with a smoke test, so the day they run smoothly in all of our integrations, we get this for free.